content/documents/ordercloud-supported-regions.mdx

```diff
@@ -0,0 +1,28 @@
+---
+type:           article
+title:          OrderCloud Supported Regions
```

Comment:
suggestion: I think it would be clearer to name this "OrderCloud Supported Regions and Environments"

It may also be helpful to link to this article in the existing https://ordercloud.io/knowledge-base/ordercloud-environments and https://ordercloud.io/knowledge-base/ordercloud-regions articles

Reply:
@erincdustin done, thank you for the suggestions!